api/src/test/java/org/openmrs/module/fhir2/providers/r4/AllergyIntoleranceFhirResourceProviderTest.java

```diff
@@ -336,4 +339,51 @@ public void createAllergy_shouldCreateAllergyIntolerance() {
 		assertThat(result.getResource(), CoreMatchers.equalTo(allergyIntolerance));
 	}
 	
+	@Test
+	public void shouldUpdateAllergy() {
```

Comment:
@Akayeshmantha thanks for the work,,,,,
may be the naming can be like this `updateAllergy_shouldUpdateAllergyIntolerance()`

Reply:
Good point @gitcliff